react/features/subtitles/components/LanguageListItem.tsx

```diff
@@ -0,0 +1,79 @@
+// @ts-ignore
+import { makeStyles } from '@material-ui/styles';
+import React, { useCallback } from 'react';
+
+// @ts-ignore
+// eslint-disable-next-line import/order
+import { translate } from '../../base/i18n';
+
+// @ts-ignore
+import { Icon } from '../../base/icons/components';
+import { IconCheck } from '../../base/icons/svg/index';
+import { Theme } from '../../base/ui/types';
+
+interface ILanguageListItemProps {
+
+    /**
+     * Whether or not the button should be full width.
+     */
+    lang: string,
+
+    /**
+     * Click callback.
+     */
+    onLanguageSelected: (lang: string) => void;
+
+    /**
+     * The id of the button.
+     */
+    selected?: boolean;
+
+    t: Function,
```

Comment:
I saw that we can use `extends WithTranslation {` and when we do this, than this line can be removed as t will be present by extending WithTranslation

Reply:
done